.github/workflows/jacoco_check.yml

```
@@ -33,7 +33,7 @@ jobs:
           # The codebase for all Scala versions is the same, so the coverage is calculated only once
           # Scala 2.12 is chosen since it is supported by the most wide range of Spark versions and
           # vendor distributions.
-          - scala: 2.12.12
+          - scala: 2.12.18
```

```suggestion
          - scala: 2.12
```

I don't think there is a need for a full scala version here, only the short one is needed and all the long formats can be transformed into short ones.

---

There is `scalaShort` below that we are using for some purposes of having just major+minor versions, but to be fair I don't care that much, whatever works - I borrowed this from https://github.com/AbsaOSS/spark-data-standardization/blob/master/.github/workflows/jacoco_check.yml and checked a few other projects as well, so I haven't really implemented this by myself